### PR TITLE
Tell IANA not to encode the RDAP base URL response

### PR DESCRIPTION
**Pull request by gbrodman**
tested on alpha (which I should have done before), our request header tells IANA to gzip the data but they're using Brotli instead. Instead of trying to worry about decoding whatever they happen to send, just don't encode it.

---
This change is [Reviewable](https://reviewable.io/reviews/google/nomulus/1705)

### CODE REVIEW

**Review by mindhog — CHANGES_REQUESTED** [review]
> Reviewed 2 of 2 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1705)* status: all files reviewed, 1 unresolved discussion (waiting on @gbrodman)

___
*[`core/src/main/java/google/registry/rdap/UpdateRegistrarRdapBaseUrlsAction.java` line 102 at r1](https://reviewable.io/reviews/google/nomulus/1705#-N6saGNK2-mjXVw86XJ1:-N6saGNL-ZQIN8IvQptd:bwdnt8r) ([raw file](https://github.com/google/nomulus/blob/09e0d85721d1b702bc67e759167c963540427d34/core/src/main/java/google/registry/rdap/UpdateRegistrarRdapBaseUrlsAction.java#L102)):*
> ```Java
>     try {
>       HttpRequest request = httpTransport.createRequestFactory().buildGetRequest(RDAP_IDS_URL);
>       // IANA can return weird encodings, like br even if we ask for gzip
> ```

This actually isn't an IANA thing: it's an App Engine thing.  GAE's HTTP client proxy appends "br" to the accept encoding header regardless of what you specify.  We never managed to resolve this for uploads, we just stopped examining the response payload.

In short, I don't think this will work :-/ 



_Code quote:_
```Java
like br even if we ask for gzip
```

**Review by gbrodman — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1705)* status: 1 of 2 files reviewed, 1 unresolved discussion (waiting on @mindhog)

___
*[`core/src/main/java/google/registry/rdap/UpdateRegistrarRdapBaseUrlsAction.java` line 102 at r1](https://reviewable.io/reviews/google/nomulus/1705#-N6saGNK2-mjXVw86XJ1:-N6sfDUN1pCv7q7-C2HI:bqwprth) ([raw file](https://github.com/google/nomulus/blob/09e0d85721d1b702bc67e759167c963540427d34/core/src/main/java/google/registry/rdap/UpdateRegistrarRdapBaseUrlsAction.java#L102)):*
<details><summary><i>Previously, mindhog (Michael Muller) wrote…</i></summary><blockquote>

This actually isn't an IANA thing: it's an App Engine thing.  GAE's HTTP client proxy appends "br" to the accept encoding header regardless of what you specify.  We never managed to resolve this for uploads, we just stopped examining the response payload.

In short, I don't think this will work :-/ 
</blockquote></details>

This works on Alpha, as mentioned in the description

**Review by gbrodman — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1705)* status: 1 of 2 files reviewed, 1 unresolved discussion (waiting on @mindhog)

___
*[`core/src/main/java/google/registry/rdap/UpdateRegistrarRdapBaseUrlsAction.java` line 102 at r1](https://reviewable.io/reviews/google/nomulus/1705#-N6saGNK2-mjXVw86XJ1:-N6sfQeR8TRAN7PsfpCe:buzfg9c) ([raw file](https://github.com/google/nomulus/blob/09e0d85721d1b702bc67e759167c963540427d34/core/src/main/java/google/registry/rdap/UpdateRegistrarRdapBaseUrlsAction.java#L102)):*
<details><summary><i>Previously, gbrodman wrote…</i></summary><blockquote>

This works on Alpha, as mentioned in the description 
</blockquote></details>

maybe the GAE proxy only adds "br" to the header if it exists with some content already?

**Review by mindhog — APPROVED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1705)* status: 1 of 2 files reviewed, all discussions resolved (waiting on @mindhog)

___
*[`core/src/main/java/google/registry/rdap/UpdateRegistrarRdapBaseUrlsAction.java` line 102 at r1](https://reviewable.io/reviews/google/nomulus/1705#-N6saGNK2-mjXVw86XJ1:-N6sgo5a9MJJ3UTbo3ix:b8nvlir) ([raw file](https://github.com/google/nomulus/blob/09e0d85721d1b702bc67e759167c963540427d34/core/src/main/java/google/registry/rdap/UpdateRegistrarRdapBaseUrlsAction.java#L102)):*
<details><summary><i>Previously, gbrodman wrote…</i></summary><blockquote>

maybe the GAE proxy only adds "br" to the header if it exists with some content already? 
</blockquote></details>

Discussed offline.  I think you're correct.

**Review by gbrodman — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/google/nomulus/1705)* status: 1 of 2 files reviewed, all discussions resolved (waiting on @mindhog)

___
*[`core/src/main/java/google/registry/rdap/UpdateRegistrarRdapBaseUrlsAction.java` line 102 at r1](https://reviewable.io/reviews/google/nomulus/1705#-N6saGNK2-mjXVw86XJ1:-N6sjbOg4cenFS_nVEs0:b-kz3iwu) ([raw file](https://github.com/google/nomulus/blob/09e0d85721d1b702bc67e759167c963540427d34/core/src/main/java/google/registry/rdap/UpdateRegistrarRdapBaseUrlsAction.java#L102)):*
<details><summary><i>Previously, mindhog (Michael Muller) wrote…</i></summary><blockquote>

Discussed offline.  I think you're correct.
</blockquote></details>

https://b.corp.google.com/issues/112388677 has some more info, for posterity
